test/recipes/80-test_ssl_old.t

```diff
@@ -606,14 +606,14 @@ sub testssl {
             ok(run(test([@ssltest, "-bio_pair", "-tls1", "-cipher", "PSK", "-psk", "abc123"])),
                'test tls1 with PSK via BIO pair');
           }
+	}
 
         SKIP: {
             skip "skipping auto DH PSK tests", 1
-                if ($no_dh || $no_psk);
+                if ($no_dh || $no_psk || $no_tls1_2);
 
-            ok(run(test(['ssl_old_test', '-psk', '0102030405', '-cipher', '@SECLEVEL=2:DHE-PSK-AES128-CCM'])),
+            ok(run(test(['ssl_old_test', '-client_max_proto', 'tls1.2', '-psk', '0102030405', '-cipher', '@SECLEVEL=2:DHE-PSK-AES128-CCM'])),
                'test auto DH meets security strength');
-          }
 	}
```

Comment:
Could you please fix the indentation in this block?

Reply:
will do.

Reply:
better now?
I also changed the indentation in #18252 accordingly.